Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG MINICONDA_VERSION
-FROM docker.lco.global/docker-miniconda3:${MINICONDA_VERSION}
```

Comment:
Was this supposed to get committed?

Reply:
It was not, but nice rhetorical question. I've updated it.